server/plugin.go

```diff
@@ -20,9 +23,53 @@ type Plugin struct {
 	configuration *configuration
 }
 
-// ServeHTTP demonstrates a plugin that handles HTTP requests by greeting the world.
+// ServeHTTP demonstrates a plugin that handles HTTP requests.
 func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
-	fmt.Fprint(w, "Hello, world!")
+	cookie, err := r.Cookie("canary")
+	if err != nil {
+		p.API.LogInfo("Canary cookie does not exist. Setting up.")
```

Comment:
We should probably switch these to debug messages and also include the user ID which you can get like this: https://github.com/mattermost/mattermost-plugin-github/blob/master/server/api.go#L77

Reply:
Thanks for the suggestion. Done ;)